posthog/api/dashboard.py

```diff
@@ -104,6 +105,16 @@ def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> Dashboard:
                     team=team,
                 )
 
+        # Manual tag creation since this create method doesn't call super()
+        if self.is_licensed():
+            if self.is_licensed() and self.initial_data.get("tags", None):
+                try:
+                    from ee.api.ee_tagged_item import EnterpriseTaggedItemSerializerMixin
+                except ImportError:
+                    pass
+                else:
+                    EnterpriseTaggedItemSerializerMixin(self).set_tags(self.initial_data["tags"], dashboard)
+
```

Comment:
1. Nope only on dashboards since we don't called the super() method on the create() method. It's annoying that we don't do this only on dashboards which means we have to write manual code to loop super initialization in. Not sure what the reasoning behind this decision was but seems like tech debt.
2. Thanks good catch!
3.